source/use_cases/aws-restaurant-management-demo/lib/lambda/manager/check-late-orders/index.js

```diff
@@ -18,49 +18,41 @@ const sns = new aws.SNS();
 
 // Handler
 exports.handler = async (event) => {
+
+  // Any order created more than LATE_ORDER_THRESHOLD minutes ago
+  // that is still open is overdue
+  const lateInterval = Number(process.env.LATE_ORDER_THRESHOLD) * 60000;
+  const lateThreshold = Number(new Date().getTime()) - lateInterval;
     
   // Setup the parameters
-  const currentTime = new Date().getTime();
   const params = {
-    TableName: process.env.DDB_TABLE_NAME
+    KeyConditionExpression:
+      "gsi1pk = :type and gsi1sk < :sortEnd",
```

Comment:
Is this file part of the Database handler Lambda Layer? Because that's the only place I would expect to see code like this.